### ISSUE TITLE
Panic on import "google.golang.org/grpc"

### ISSUE THREAD

**Issue by zliang-min** [traefik/yaegi#260]
Steps:
1. run `yaegi`
2. run `import "google.golang.org/grpc"`

It then will panic:

```console
> import "google.golang.org/grpc"
/Users/gimi/workspace/go/src/golang.org/x/net/context/go17.go:15:15: panic
panic: reflect.Set: value of type context.Context is not assignable to type uint32 [recovered]
        panic: reflect.Set: value of type context.Context is not assignable to type uint32

goroutine 1 [running]:
github.com/containous/yaegi/interp.runCfg.func1(0xc0000eb590, 0xc0003b3d00)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/run.go:107 +0x1e9
panic(0x17fcf20, 0xc00038d200)
        /usr/local/Cellar/go/1.12.7/libexec/src/runtime/panic.go:522 +0x1b5
reflect.Value.assignTo(0x18644c0, 0xc00038d1f0, 0x94, 0x191e2a9, 0xb, 0x17fd1a0, 0x0, 0xc000364f00, 0x1, 0x1)
        /usr/local/Cellar/go/1.12.7/libexec/src/reflect/value.go:2339 +0x437
reflect.Value.Set(0x17fd1a0, 0xc00031c8c0, 0x18a, 0x18644c0, 0xc00038d1f0, 0x94)
        /usr/local/Cellar/go/1.12.7/libexec/src/reflect/value.go:1473 +0xa8
github.com/containous/yaegi/interp.callBin.func5(0xc0000eb590, 0xc0003660f0)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/run.go:838 +0x269
github.com/containous/yaegi/interp.runCfg(0xc0003b3d00, 0xc0000eb590)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/run.go:112 +0x70
github.com/containous/yaegi/interp.(*Interpreter).run(0xc0000c4540, 0xc0003b3200, 0x0)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/run.go:93 +0x126
github.com/containous/yaegi/interp.(*Interpreter).importSrcFile(0xc0000c4540, 0x0, 0x0, 0xc0000e71e1, 0x18, 0xc0000e71f2, 0x7, 0xc000364160, 0xc0001a2ca8)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/src.go:95 +0x5c9
github.com/containous/yaegi/interp.(*Interpreter).gta.func1(0xc000368a00, 0xc0001a2fb0)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/gta.go:141 +0xc5e
github.com/containous/yaegi/interp.(*node).Walk(0xc000368a00, 0xc0001a2fb0, 0x0)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/interp.go:106 +0xc9
github.com/containous/yaegi/interp.(*node).Walk(0xc00034bb00, 0xc0001a2fb0, 0x0)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc00034b900, 0xc0001a2fb0, 0x0)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*Interpreter).gta(0xc0000c4540, 0xc00034b900, 0xc0000feed0, 0x22, 0xc0000feed0, 0x22)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/gta.go:17 +0xf4
github.com/containous/yaegi/interp.(*Interpreter).importSrcFile(0xc0000c4540, 0xc0000e6960, 0x16, 0xc0000fe541, 0x22, 0xc0000fe558, 0xb, 0x0, 0x0)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/src.go:68 +0xade
github.com/containous/yaegi/interp.(*Interpreter).gta.func1(0xc000260c00, 0xc0001a3610)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/gta.go:141 +0xc5e
github.com/containous/yaegi/interp.(*node).Walk(0xc000260c00, 0xc0001a3610, 0x0)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/interp.go:106 +0xc9
github.com/containous/yaegi/interp.(*node).Walk(0xc000260100, 0xc0001a3610, 0x0)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc000225f00, 0xc0001a3610, 0x0)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*Interpreter).gta(0xc0000c4540, 0xc000225f00, 0xc0000e6960, 0x16, 0xc0000e6960, 0x16)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/gta.go:17 +0xf4
github.com/containous/yaegi/interp.(*Interpreter).importSrcFile(0xc0000c4540, 0x0, 0x0, 0xc0000e62e1, 0x16, 0xc0000e62f3, 0x4, 0x0, 0x8)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/src.go:68 +0xade
github.com/containous/yaegi/interp.(*Interpreter).gta.func1(0xc000107e00, 0x18d9520)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/gta.go:141 +0xc5e
github.com/containous/yaegi/interp.(*node).Walk(0xc000107e00, 0xc0001a3c70, 0x0)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/interp.go:106 +0xc9
github.com/containous/yaegi/interp.(*node).Walk(0xc000107d00, 0xc0001a3c70, 0x0)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc000107b00, 0xc0001a3c70, 0x0)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*Interpreter).gta(0xc0000c4540, 0xc000107b00, 0xc0000e27d8, 0x4, 0x0, 0xc0000e27d8)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/gta.go:17 +0xf4
github.com/containous/yaegi/interp.(*Interpreter).Eval(0xc0000c4540, 0xc0000e62a0, 0x20, 0xc0001a3df0, 0x1f, 0x1914095, 0x1, 0xc0000e62a0)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/interp.go:252 +0x104
github.com/containous/yaegi/interp.(*Interpreter).Repl(0xc0000c4540, 0xc0000be000, 0xc0000c2000)
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/interp/interp.go:332 +0x1c3
main.main()
        /Users/gimi/workspace/go/pkg/mod/github.com/containous/yaegi@v0.0.1/cmd/yaegi/yaegi.go:85 +0x4e2
```

**Comment by ldez** [comment]
note for later: the build  tags do not seem to have been read (`/go/src/golang.org/x/net/context/go17.go`)

**Comment by mvertes** [comment]
This panic is now fixed since f7810d976145b314f25a3c1906d019995c6f7608 and now import of package fails because a dependency on `unsafe`. Proposal #372 underlines a solution for that.

**Comment by mvertes** [comment]
Closing in favor of #1464, almost identical, but with a better and more recent reproducible case.
